vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/LazyLoadingIT.java

```diff
@@ -96,85 +96,6 @@ public void selectItem_changeFilter_properlyFilteredItems() {
         assertNotRendered("Item 2");
     }
 
-    @Test
```

Comment:
Is it safe to remove these tests? Are all the special / edge cases from the old tests covered in the new tests?

Reply:
Updated the tests so that they also check how many items are loaded before reaching the limit.